### PR TITLE
Add usage docs to README.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
This PR adds a README to this repo, with usage examples.

### CODE REVIEW

**Review by msporny — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,99 @@
-# lru-memoize
\ No newline at end of file
+# LRU-Memoize _(@digitalbazaar/lru-memoize)_
+
+[![Node.js CI](https://github.com/digitalbazaar/lru-memoize/workflows/Node.js%20CI/badge.svg)](https://github.com/digitalbazaar/lru-memoize/actions?query=workflow%3A%22Node.js+CI%22)
+[![NPM Version](https://img.shields.io/npm/v/@digitalbazaar/lru-memoize.svg)](https://npm.im/@digitalbazaar/lru-memoize)
+
+> A Memoized wrapper around the javascript [`lru-cache`](https://www.npmjs.com/package/lru-cache) library.
```
Comment (davidlehn) [inline README.md:6]:
```suggestion
> A Memoized wrapper around the JavaScript [`lru-cache`](https://www.npmjs.com/package/lru-cache) library.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,99 @@
...
+```js
+import { LruCache } from '@digitalbazaar/lru-memoize';
+
+// Cache expiration/TTL: 5 seconds
+const myCache = new LruCache({ maxAge: 5000 });
+
+async function fetchStatus() {
+  // simulate an async task
+  await delay(100);
+  executedTestFn = true;
```
Comment (davidlehn) [inline README.md:70]: Maybe need a comment to explain what this is for?

File: README.md [hunk 3]
```diff
@@ -1 +1,99 @@
...
+
+- [Background](#background)
+- [Install](#install)
+- [Usage](#usage)
+- [Contribute](#contribute)
+- [License](#license)
+
+## Background
+
+lru-memoize is used to
```
Comment (davidlehn) [inline README.md:18]:
```suggestion
`lru-memoize` is used to
```

**Review by dlongley — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (dmitrizagidulin) [reply README.md:70]: It isn't really for anything; the example was just cut & paste'd from the unit tests.
In any case, I propose we handle the explanation in another issue / PR :)

**Review by davidlehn — APPROVED** [review]

**Pull request merged**